### PR TITLE
Add milestone 2 report to project page

### PR DESCRIPTION
**Pull request by cjdelisle**
Hello folks, I know things have been a bit quiet publicly but work has been ongoing with this project in order to get it complete. Please review this final report so that we can bring this project to a close.

### CODE REVIEW

File: projects/2019_11_13_pkt_insight.md [hunk 1]
```diff
@@ -329,3 +329,78 @@ Second payment has been partially made in transactions:
...
+  * The evaluators of the proposal are invited to setup an instance to verify that it can be easily installed.
+* [x] **S_FEATURES** At least 90% of the described features achieved
+  * In this milestone, there were 12 features specified and all but 1 were delivered.
+
+### Request for guidance
+
+In addition to the applicant's request that the project be accepted as complete, the applicant additionally
+requests guidance on the following question:
+
+Should pkt-explorer-backend, pkt-explorer-insightfe or pkt-explorer be transferred to the pkt-cash namespace?
```
Comment (benhylau) [inline projects/2019_11_13_pkt_insight.md:403]: +1

File: projects/2019_11_13_pkt_insight.md [hunk 2]
```diff
@@ -329,3 +329,78 @@ Second payment has been partially made in transactions:
...
+
+If the answer is yes, the applicant requests continued administrative access to the repositories in order
```
Comment (benhylau) [inline projects/2019_11_13_pkt_insight.md:405]: +1 as collaborator on those repos sound good?

File: projects/2019_11_13_pkt_insight.md [hunk 3]
```diff
@@ -329,3 +329,78 @@ Second payment has been partially made in transactions:
...
+### Success criteria
+
+* [x] **S_CI** Codebase passing automated continuous integration build
+  * https://travis-ci.org/github/cjdelisle/pkt-explorer-backend
+  * https://travis-ci.org/github/cjdelisle/pkt-explorer
+* [ ] **S_INSTALL** At least one successful independent installation of the explorer
+  * There was previously an installation of the explorer at https://pkt.hprivakos.net/ but currently this is down.
+  * The evaluators of the proposal are invited to setup an instance to verify that it can be easily installed.
+* [x] **S_FEATURES** At least 90% of the described features achieved
+  * In this milestone, there were 12 features specified and all but 1 were delivered.
```
Comment (benhylau) [inline projects/2019_11_13_pkt_insight.md:396]: Add a LICENSE to repo?

**Comment by benhylau** [comment]
Congrats it's looking really good :)

File: projects/2019_11_13_pkt_insight.md (hunk 2 above)
Comment (cjdelisle) [reply projects/2019_11_13_pkt_insight.md:405]: That sounds like a good approach

File: projects/2019_11_13_pkt_insight.md (hunk 3 above)
Comment (cjdelisle) [reply projects/2019_11_13_pkt_insight.md:396]: I'll do that later this evening, FTR it's all MIT

Comment (cjdelisle) [reply projects/2019_11_13_pkt_insight.md:396]: Done
https://github.com/cjdelisle/pkt-explorer/blob/master/LICENSE.md
https://github.com/cjdelisle/pkt-explorer-backend/blob/master/LICENSE.md (already existed)

File: projects/2019_11_13_pkt_insight.md (hunk 1 above)
Comment (Arceliar) [reply projects/2019_11_13_pkt_insight.md:403]: Agreed

File: projects/2019_11_13_pkt_insight.md (hunk 2 above)
Comment (Arceliar) [reply projects/2019_11_13_pkt_insight.md:405]: Agreed

File: projects/2019_11_13_pkt_insight.md (hunk 1 above)
Comment (neilalexander) [reply projects/2019_11_13_pkt_insight.md:403]: Yep I agree.

File: projects/2019_11_13_pkt_insight.md (hunk 2 above)
Comment (neilalexander) [reply projects/2019_11_13_pkt_insight.md:405]: Yep, agreed.

File: projects/2019_11_13_pkt_insight.md (hunk 1 above)
Comment (backupbrain) [reply projects/2019_11_13_pkt_insight.md:403]: Yes

File: projects/2019_11_13_pkt_insight.md (hunk 2 above)
Comment (backupbrain) [reply projects/2019_11_13_pkt_insight.md:405]: Yes

**Comment by cjdelisle** [comment]
Merging because it is agreed

**Pull request merged**